srcpkgs/docker-machine-driver-kvm/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'docker-machine-driver-kvm'
+pkgname=docker-machine-driver-kvm
+version=0.10.1
+revision=1
+archs="x86_64 aarch64" # Broken on 32bit archs
```

Comment:
There are other 64 bit architectures out there (ppc64 comes to mind) and it should also work on musl with 64 bit architectures. It would be better to turn that condition around and explicitely mark the package as broken on 32 bit architectures which can be done by checking `XBPS_WORDSIZE`.